helm/alfresco-content-services/values.yaml

```diff
@@ -469,6 +468,8 @@ postgresql:
   enabled: true
   nameOverride: postgresql-acs
   imageTag: "11.7.0"
+  commonAnnotations:
```

Comment:
Do you know why we need to add this?

Reply:
it's an harmless fake placeholder, there is an error in the postgresql chart where it still prints the annotations: empty value in ingress and services which does not validate, so in case you have a kubernetes admission controller active which checks for validation the chart does not deploy, with this change the output file validates fully

Reply:
OK, thanks.